pkgs/applications/backup/proxmox-backup-client/default.nix

```diff
@@ -1,24 +1,25 @@
 {
-  lib, fetchgit, rustPlatform, pkg-config, openssl, fuse3, libuuid, acl,
-  libxcrypt, git, installShellFiles, sphinx, stdenv,
+  lib, fetchgit, rustPlatform, pkg-config, openssl, fuse3, libuuid,
+  acl, libxcrypt, git, installShellFiles, sphinx, stdenv, fetchpatch,
+  testers, proxmox-backup-client,
```

Comment:
since you are allready adding new inputs, maybe reformat them?
```nix
{ lib
, fetchgit
, ...
}
```

Reply:
Will do, thanks!